base_geoengine/geo_operators.py

```diff
@@ -196,7 +196,7 @@ def get_geo_equal_sql(self, table, col, value, rel_col=None,
         else:
             base = self.geo_field.entry_to_shape(value, same_type=False)
             compare_to = "ST_GeomFromText('%s')" % (base.wkt,)
-        return " %s.%s = %s" % (table, col, compare_to)
+        return " _%s_.%s = %s" % (table, col, compare_to)
```

Comment:
No sure which one of those function is breaking the queries with demo module installed.

Reply:
ping @tarteo in case you missed this.

The current PR is breaking geoengine, I cannot merge it.